### DISCUSSION TITLE
Please make the mica work to taskbar and start

### DISCUSSION THREAD

**Discussion by moondling** [MicaForEveryone/MicaForEveryone#160] [Ideas]
This app is great, however i still dont use transparency effects usually because i cant stand the acrylic in taskbar and start, its too inconsistent and bright. Is there anyway to affect them with mica too? 

Thanks

**Comment by dongle-the-gadget** [comment]
XAML Diagnostics is required, so the current way we do it won't work.

**Comment by xmine64** [comment]
Use TranslucentTB or TaskbarX or ExplorerPatcher or whatever

**Reply by dongle-the-gadget** [reply]
The new system tray updates broke TTB.

(smh email)
